sdk/Dagger.SDK.SourceGenerator/Dagger.SDK.SourceGenerator/Code/CodeRenderer.cs

```diff
@@ -54,34 +55,39 @@ public string RenderEnum(Type type)
         return $$"""
                  {{RenderDocComment(type)}}
                  [JsonConverter(typeof(JsonStringEnumConverter<{{type.Name}}>))]
-                 public enum {{type.Name}} {
+                 public enum {{type.Name}} 
+                 {
                      {{string.Join(",", evs)}}
                  }
                  """;
     }
 
     public string RenderInputObject(Type type)
     {
-        var properties = type.InputFields.Select(field => $$"""
-                                                            {{RenderDocComment(field)}}
-                                                            public {{field.Type.GetTypeName()}} {{Formatter.FormatProperty(field.Name)}} { get; } = {{field.GetVarName()}};
-                                                            """);
+        var properties = type.InputFields.Select(field => 
+            $$"""
+            {{RenderDocComment(field)}}
+            public {{field.Type.GetTypeName()}} {{Formatter.FormatProperty(field.Name)}} { get; } = {{field.GetVarName()}};
+            """);
 
         var constructorFields =
             type.InputFields.Select(field => $"""{field.Type.GetTypeName()} {field.GetVarName()}""");
 
-        var toKeyValuePairsProperties = type.InputFields.Select(field => $"""
-                                                                          kvPairs.Add(KeyValuePair.Create("{field.Name}", {RenderArgumentValue(field, asProperty: true)} as Value));
-                                                                          """);
-
-        var toKeyValuePairsMethod = $$"""
-                                      public List<KeyValuePair<string,Value>> ToKeyValuePairs()
-                                      {
-                                          var kvPairs = new List<KeyValuePair<string, Value>>();
-                                          {{string.Join("\n", toKeyValuePairsProperties)}}
-                                          return kvPairs;
-                                      }
-                                      """;
+        var toKeyValuePairsProperties = type.InputFields.Select(field => 
+            $"""
+            kvPairs.Add(KeyValuePair.Create("{field.Name}", {RenderArgumentValue(field, asProperty: true)} as Value));
+            """);
+
+        var toKeyValuePairsMethod = 
+              $$"""
```

Comment:
Is this happens by formatter or you just change the line manually?

Reply:
This was me, but actually .editorconfig didn't say anything.

Reply:
I see.